docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "3.3"
+
+services:
+  db:
+    image: mysql:5.7
+    volumes:
+      - data:/var/lib/mysql
+    environment:
+      MYSQL_DATABASE: prestashop
+      MYSQL_USER: prestashop
+      MYSQL_PASSWORD: prestashop
+      MYSQL_ROOT_PASSWORD: prestashop
+    ports:
+      - 3308:3306
+    container_name: local-prestashop-mysql
+
+  prestashop:
+    depends_on:
+      - db
+    image: prestashop/prestashop:1.7
+    ports:
+      - 9000:80
+      - 443:443
+    environment:
+      PS_DB_HOST: db:3306
+      PS_DB_USER: prestashop
+      PS_DB_PASSWORD: prestashop
+    volumes:
+      - ./html:/var/www/html
+    container_name: local-web-prestashop
+
+  phpmyadmin:
+    depends_on:
+      - db
+    image: phpmyadmin/phpmyadmin
+    ports:
+      - 9001:80
+    environment:
+      PMA_HOST: db
+    container_name: local-phpmyadmin
```

Comment:
I'll remove this container. It's better to have a database client, like [dbeaver](https://dbeaver.io/) in our hosts, and use throughout all databases

Reply:
No problem, I have already removed it in the next commit